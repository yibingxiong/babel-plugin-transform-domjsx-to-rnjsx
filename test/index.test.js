const babel = require('@babel/core');
const plugin = require('../src/index');
const fs = require('fs');
const path = require('path');

var example = fs.readFileSync(path.join(__dirname, '__fixtures__', 'changed.js'), {encoding: 'utf8'});

it('works', () => {
  const {code} = babel.transform(example, {plugins: [plugin]});
  expect(code).toMatchSnapshot();
});