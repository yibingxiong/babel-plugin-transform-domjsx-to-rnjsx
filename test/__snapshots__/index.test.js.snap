// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`works 1`] = `
"function b() {
  console.log(1);
}

function A() {
  return <View View={b}>
            <View>
                {[1, 2, 3, 4].map(item => {
        return <View>item</View>;
      })}
            </View>
        </View>;
}"
`;
